breathecode/payments/urls.py

```diff
@@ -29,6 +30,10 @@
     path('me/service/consumable', MeConsumableView.as_view(), name='me_service_consumable'),
     path('me/subscription', MeSubscriptionView.as_view()),
     path('me/subscription/<int:subscription_id>', MeSubscriptionView.as_view()),
+    path('me/subscription/<int:subscription_id>', MeSubscriptionChargeView.as_view()),
+    path('me/planfinancing', MeSubscriptionView.as_view()),
+    path('me/planfinancing/<int:plan_financing_id>', MeSubscriptionView.as_view()),
```

Comment:
Why is this url repeated but with different views, how will it determine which view it should take